doc/classes/Node.xml

```diff
@@ -927,10 +927,10 @@
 		<constant name="NOTIFICATION_WM_DPI_CHANGE" value="1009">
 		</constant>
 		<constant name="NOTIFICATION_VP_MOUSE_ENTER" value="1010">
-			Notification received when the mouse enters the viewport.
+			Notification received when the mouse cursor enters the [Viewport]'s visible area, that is not concealed behind other [Control]s or [Window]s, provided its [member Viewport.gui_disable_input] is [code]false[/code] and regardless if it's currently focused or not.
```

Comment:
Can a Control obscure a Viewport?
btw you seem to use `[]` inconsistently compared to descriptions above.

Reply:
With the latest update, I use [] consistently in the docs.